doc/api/core/pointsets.rst

```diff
@@ -51,6 +51,19 @@ data:
    demos.plot_datasets('StructuredGrid')
 
 
+The :class:`pyvista.PointSet` (as of ``vtk==9.1.0``) is a concrete class for
+storing a set of points.
+
+.. jupyter-execute::
+   :hide-code:
+
+   import numpy as np
+   import pyvista
+   rng = np.random.default_rng(0)
+   points = rng.random((10, 3))
+   pset = pyvista.PointSet(points)
+   pset.plot()
+
```

Comment:
I don't see changes here being reflected in the built docs. No idea what's going on. Can you check by downloading the doc build artifact? The other changes are visible.

Reply:
Looking into this.

Reply:
Local build works fine, might be a CI fluke. Probably shouldn't hold this PR up.